concepts/modularization/README.md

```diff
@@ -41,13 +41,47 @@ If the API you need (like a core Kubernetes API or Istio virtual service) is not
 # Release channels
 Release channels let customers try new modules and features early, and decide when the updates should be applied. 
 
+The first use case will be modeled as the alpha channel. Modules available in the alpha channel are developed with all quality measures in place (functional correctness, security, etc.), but they might still have unstable API or be changed without keeping backward compatibility. When you use a module from the alpha channel, you won't get full SLA guarantees for that module or other modules that are affected (directly or indirectly).
+
+The second use case (deciding when updates should be applied) will require 2 production-grade channels with a different update schedule. The fast channel will get updates as soon as they are released and have passed all quality gates. The regular channel will get updates a few days later. Customers can switch the entire cluster or a particular component to the regular channel to check if the upstream changes do not cause any issues with their workload. Changing back to the regular channel is possible, but the module version won't be downgraded - the next version has to reach the channel to be applied.
+
+Moving module versions between channels should be automated as much as possible using a promotion strategy suitable for the particular version change (major/minor/patch) or priority (regular/hot-fix).
+
 ![](assets/release-channels.drawio.svg)
 
-The first use case will be modeled as the alpha channel. Modules available in the alpha channel are developed with all quality measures in place (functional correctness, security, etc.), but they might still have unstable API or be changed without keeping backward compatibility. When you use a module from the alpha channel, you won't get full SLA guarantees for that module or other modules that are affected (directly or indirectly).
+In the diagram, we have an example with 3 modules that got new releases. Each one represents a different promotion strategy:
+
+**Module X**
+
+Module `x` is in active development. Version 1.0.x is out for a longer time (version 1.0.18 in the regular channel). The new version (1.1.0) was released recently and is available in the fast channel for a few days. The new version is not super stable yet. The patch version (1.1.1) with bug fixes was required a few days later. Also, the security vulnerability was fixed in that patch but the version is too fresh to go to the regular channel already, therefore new patch release for the 1.0.x version is required (only the security patch included). The patch goes only to the regular channel as we can't downgrade versions in the alpha or fast channel. 
```

Comment:
This patching to the regular channel is already an exception to our timed flow of 2 weeks incubation in the fast channel. I think we should judge carefully here since this would mean we have to introduce release branches.

Reply:
Release branches are there and are inevitable. It is more important to use them only for hot-fixes, not for regular development. If we do not have anything critical the next version from main branch should bring bug fixes and security patches.